### DISCUSSION TITLE
Drivers for HX8347D

### DISCUSSION THREAD

**Discussion by adolfonovo** [Bodmer/TFT_eSPI#1711] [New processor/display support or capability request]
Hi @Bodmer 
is there any chance to include this driver?...
Thanks and congrats for your job

**Comment by Anxodia** [comment]
Hi,

I would also be happy if it gets HX8357D support, I like to use it with my Mega2560.

**Comment by Bodmer** [comment]
Can you provide links to seller pages. SPI or parallel?

**Comment by adolfonovo** [comment]
Yes sure!! The display operates in 8bit parallel mode.

AZDelivery 2.4 pulgadas TFT LCD Touch Display Modulo Shield SPI XPT2046
240X320 pixeles ILI9341 5V 3.3V compatible con Arduino con E-Book incluido!
https://www.amazon.es/dp/B086W6D9Z2/ref=cm_sw_r_cp_api_glt_i_6J1BDC396ZDZWRE755DJ?_encoding=UTF8&psc=1

El El lun, 13 sept 2021 a las 13:16, Bodmer ***@***.***>
escribió:

> Can you provide links to seller pages. SPI or parallel?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Bodmer/TFT_eSPI/issues/1315#issuecomment-918091711>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARTH5M4EFG3HKJKBW6T3LG3UBXMSNANCNFSM5DBPI3MA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 

*Adolfo Novo*

<https://www.linkedin.com/in/adolfonovo/>
